Loan.ts

```diff
@@ -0,0 +1,32 @@
+import FineractAPI from './FineractAPI';
+
+export interface LoanConfig{
+    offset?:Number,
+    limit?:Number,
+    orderBy?:String,
+    sortBy?:String,
+    officeId?:Number,
+    underHierarchy?:String,
+    accountNo?:String,
+    externalId?:String,
+    sqlSearch?:String
+}
+export default class Loan{
+
+fineract_obj:FineractAPI
+constructor(fineract_obj:FineractAPI)
+{
+   this.fineract_obj=fineract_obj;
+}
+
+list_loan(config:LoanConfig):any {
```

Comment:
add async keyword for this function


```suggestion
async list_loan(config:LoanConfig):any {
```